### PR TITLE
[docs] How to build runtime tizen gbs rpi4

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce how to build runtine in GBS for Tizen/AARCH64 and run in RPi4

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: docs/howto/how-to-build-runtime-tizen-gbs-rpi4.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+/dev/sdj10      62132224 62263295   131072   64M 83 Linux
+/dev/sdj11      62265344 62521343   256000  125M 83 Linux
+```
+
+(3) Download images
+
+Please visit http://download.tizen.org/snapshots/tizen/unified/latest/images/standard/iot-boot-arm64-rpi4/
+and http://download.tizen.org/snapshots/tizen/unified/latest/images/standard/iot-headed-3parts-aarch64-rpi.
+
+Pleae Visit `iot-boot-armv7l-rpi4` folder for ARM32 images.
```
Comment (mhs4670go) [inline docs/howto/how-to-build-runtime-tizen-gbs-rpi4.md:137]:
```suggestion
Please visit `iot-boot-armv7l-rpi4` folder for ARM32 images.
```

**Comment by hseok-oh** [comment]
@lemmaa I have a question. What is purpose of `docs/device` documentation directory?

**Comment by lemmaa** [comment]
> @lemmaa I have a question. What is purpose of `docs/device` documentation directory?

@hseok-oh , it reflects the document folder structure of the previous repo as it is, and is for organizing information by reference device. Maybe it would be better to create a folder for each device rather than one file for each device like now.

If that happens, we'd better moving this document under `docs/device/rpi4`, creating a `how-to-build-runtime-for-reference-devices.md` document here, then writing the link to `docs/device/*/how-to-build-runtime-xxx.md` in the document.

**Review by lemmaa — APPROVED** [review]
LGTM, the contents itself.

File: docs/howto/how-to-build-runtime-tizen-gbs-rpi4.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
+# How to Build Runtime with GBS for Tizen/RPi4
+
+This document describes how to build runtime with GBS for Tizen AARCH64.
+As a real example, we'll aslo describe how to prepare Tizen on Raspberry Pi 4
```
Comment (chunseoklee) [inline docs/howto/how-to-build-runtime-tizen-gbs-rpi4.md:4]:
```suggestion
As a real example, we'll aslo describe how to prepare Tizen on Raspberry Pi 4
```

**Review by chunseoklee — APPROVED** [review]

**Review by Hyunjun85 — APPROVED** [review]
LGTM

**Review by lemmaa — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**